src/main/java/com/github/nicholasmoser/gnt4/seq/ext/SeqAssembler.java

```diff
@@ -0,0 +1,1171 @@
+                    baos.write(action);
+                    //No alternative actions supported, only pure flag operations
+                    baos.write(0);
+                    baos.write(ByteUtils.fromInt32(getFlags(opcode[2], operands.replace("\"", ""))));
+                    currentOpcode = SeqHelper.getSeqOpcode(new ByteStream(baos.toByteArray()), baos.toByteArray()[0], baos.toByteArray()[1]);
+                    break;
+                case "loadTexture":
+                    //TODO
+                    // Still todo
+                    currentOpcode = SeqHelper.getSeqOpcode(new ByteStream(baos.toByteArray()), baos.toByteArray()[0], baos.toByteArray()[1]);
+                    break;
+                case "op":
+                    String firstByte = opcode[1].substring(0, 2);
+                    String secondByte = opcode[1].substring(2, 4);
+                    switch (firstByte) {
+                        case "02":
+                            switch (secondByte) {
+                                case "08" -> baos.write(UnknownOpcode.of(opcode[1], operands));
+
+                                //TODO
+                                // Still TODO
+                                default -> baos.write(UnknownOpcode.of(opcode[1], operands));
+                            }
+                            break;
+                        case "09":
+                            switch (secondByte) {
+                                case "26" -> {
+                                    baos.write(9);
+                                    baos.write(0x26);
+                                    baos.write(SEQ_RegCMD2(op1, op2));
+                                }
+                                default -> baos.write(UnknownOpcode.of(opcode[1], operands));
+                            }
+                            break;
+                        case "0E":
+                            switch (secondByte) {
+                                case "06" -> {
+                                    baos.write(0x0E);
+                                    baos.write(0x06);
+                                    baos.write(SEQ_RegCMD2(op1, op2));
+                                }
+                                default -> baos.write(UnknownOpcode.of(opcode[1], operands));
+                            }
+                            break;
+                        default:
+                            baos.write(UnknownOpcode.of(opcode[1], operands));
+                            break;
+                    }
+                    currentOpcode = new UnknownOpcode(offset, baos.toByteArray());
+                    break;
+                default:
+                    System.err.println(opcode[0]);
+                    break;
+            }
+            opcodes.add(currentOpcode);
+            offset += currentOpcode.getBytes().length;
+        }
+        for (Opcode opcode : opcodes) {
+            if (opcode instanceof BranchingOpcode) {
```

Comment:
You can change this to:

```java
if (opcode instanceof BranchingOpcode branchingOpcode) {
```

and replace `(BranchingOpcode) opcode` with `branchingOpcode`.
Same with lines 793 and 809.